### ISSUE TITLE
cf metrics store - persistent metrics in self service

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#346]
### Expected behavior
As a platform user (apps-dev /apps-ops) i want the platform to collect my cf apps metrics, and persist them + offer a standard API 

### Observed behavior
As of cf-deployment v6 / paas-templates v 40, users can get current metrics with cf cli.

### references
- see prometheus as a service issue #196 
- https://github.com/cloudfoundry/metric-store-release
- https://content.pivotal.io/blog/metric-store-a-cloud-native-time-series-database-for-cloud-foundry

**Comment by poblin-orange** [comment]
@regismarg i guess this has value for OCB (not acdc as they do not use paas-templates cf ?)

**Comment by regismarg** [comment]
@poblin-orange  : yes mistake beteween apps and services (  i want the platform to collect my cf srevices and expose them to final suer)
